### PR TITLE
More styling for Caturday-- adding poster images for cats

### PR DESCRIPTION
**Pull request by oakleywindiate**
### Description
* Summary:
1) Added CSS to create cat posters

* Where should the reviewer start:
1) App.tsx

### Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by shanekwarning — APPROVED** [review]
Amazing looks great and gives the site the vibe and UI we are looking for.

File: src/CatCards.css [hunk 1]
```diff
@@ -1,12 +1,114 @@
...
   position: relative;
   transition: all .2s ease-in-out;
-  background-color: #f8f5ec;
-  margin: 10px;
+  margin: 20px;
+  background: url(old-paper-horizontal.jpeg);
+  box-shadow: 0 0 20px 5px #000 inset;
+  padding: 2em;
+  filter: drop-shadow(-1px 6px 3px rgba(0, 0, 0, 0.5));
+  clip-path: polygon(
```
Comment (angelewilliams) [inline src/CatCards.css:10]: This is wild. So cool that you found this!

File: src/CatCards.tsx [hunk 2]
```diff
@@ -13,8 +13,9 @@ const CatCards: React.FC<CatCardProps> = ({id, img, name, key}) => {
   return (
 
     <div className="card">
+      <h1 className="wanted-poster-card">WANTED</h1>
```
Comment (angelewilliams) [inline src/CatCards.tsx:16]: Love the leaning into the western meowst wanted.

**Review by emmacbd — APPROVED** [review]
Thank you so much Oakley! This looks great.

**Review by angelewilliams — APPROVED** [review]

**Pull request merged**